script/Common/ORPHANED_LINES.txt

```diff
@@ -31,27 +34,28 @@
 [sha:8300c51663eaadb5aca8ce9e0d5f6620ee347fae]{
 -- Page -1, Offset 63.
 -- ―――よく　わからない。
--- TRANSLATION HERE
+―――I really... don't understand.
 }
 [sha:d2238e268c89185cd0b6bacad997ad3ad9cb7886]{
 -- Page -1, Offset 68.
 -- ―――ボクは子供だから　よく　わからない。
--- TRANSLATION HERE
+―――I'm just a child so... I don't really understand.
 }
 [sha:8ed6e9f0c815a2378368474ccbbb4fad56391cec]{
 -- Page -1, Offset 82.
 -- ―――<宙|ソラ>を<塞|ふさ>ぐ<天蓋|てんがい>
--- TRANSLATION HERE
+// See also 13_00_ARC13_4 offset 32647.
+―――The firmament that seals off the cosmos.
 }
 [sha:06120b440538ce306f75ae81db0b208c4770228a]{
 -- Page -1, Offset 87.
 -- ―――夜を開く<白貌|はくぼう>
--- TRANSLATION HERE
+―――A white face that opens up the night.
 }
 [sha:9195fcbdccb9e0448e48c9f138c245083f940192]{
 -- Page -1, Offset 93.
 -- ―――<幽玄|ゆうげん><麗|うら>らかに　落下する星
--- TRANSLATION HERE
+―――A profoundly beautiful falling star.
```

Comment:
Obviously 星 here refers to the Moon, so I'm unsure if its right to call it a star. Can't call it a planet either...
Maybe just using moon is right here?

> The moon, falling in profound beauty.

Reply:
I might shy away from an explicit moon here, if only because the line right above this one (when seen in game) is the 
> naught but a single moon one

and I'd like to avoid the repetition. Celestial body?